zcash_client_sqlite/src/lib.rs

```diff
@@ -882,6 +882,7 @@ pub enum FsBlockDbError {
     MissingBlockPath(PathBuf),
     InvalidBlockstoreRoot(PathBuf),
     InvalidBlockPath(PathBuf),
+    InvalidParameter(String),
```

Comment:
blocking: The intent of this particular error is that it be specifically identifiable (and recoverable). So a generic `InvalidParameter` is not the right approach, because you'd have to parse the string to identify why it failed. So this should be something explicit like `CacheMiss(BlockHeight)` so that the caller can then go and retrieve the block.

Reply:
A downside to this is that it's less clear to the caller that the problem is the `from_height` parameter, rather than "any" cache miss. So this change trades off one issue for another, but assuming that's acceptable I'm fine with going fo the option that can be matched on.